README.md

```diff
@@ -190,6 +190,51 @@ ctest --test-dir build -j
 
 For more detailed build instructions please refer to the [dedicated section on CMake](./docs/1-user-guide/build-with-cmake) or to [our web page][build_info_url].
 
+## How to CROSS-COMPILE SLEEF
+Two methods are used for cross-compiling SLEEF. Here are examples of cross-compiling SLEEF for the arm architecture on a platform with X86 architecture and Linux OS:
```

Comment:
Maybe add that both rely on existing toolchain files provided in the `toolchains/` directory.